nixos/modules/tasks/filesystems/btrfs.nix

```diff
@@ -31,5 +68,56 @@ in
       ''
         btrfs device scan
       '';
-  };
+    }
+    // mkIf cfgScrub.enable {
+      assertions = [
+        {
+          assertion = cfgScrub.enable -> (cfgScrub.fileSystems != []);
+          message = ''
+            If 'services.btrfs.autoScrub' is enabled, you need to have at least one
+            btrfs file system mounted via 'fileSystems' or specify a list manually
+            in 'services.btrfs.autoScrub.fileSystems'.
+          '';
+        }
+      ];
+
+      # This will yield duplicated units when If the user mounts a filesystem multiple times
```

Comment:
This comment seems a bit strange here: "… units when If the user …"

Reply:
Thanks for the hint!